components/todo/AddTodoCard.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="add-todo-card">
+    <form @submit.prevent="addTodo">
+      <div class="add-todo-card__body">
+        <!-- <textarea
+            v-model="form.title"
+            class="add-todo-card__input"
+            placeholder="Add Task"
+          /> -->
+        <input
+          v-model="form.title"
+          type="text"
+          class="add-todo-card__input"
+          placeholder="Add Task"
+        />
+      </div>
+      <div class="add-todo-card__footer">
+        <button type="submit" class="add-todo-card__footer__btn">
+          Add Task
+        </button>
+        <button>Delete</button>
+      </div>
+    </form>
+  </div>
+</template>
+<script>
+export default {
+  name: 'AddTodoCard',
+  data() {
+    return {
+      form: {
+        title: '',
+      },
+    }
+  },
+  methods: {
+    addTodo() {
+      this.$emit('addTodo', this.form)
+      this.form.title = ''
+    },
+  },
+}
+</script>
+
+<style scoped lang="scss">
+$card-padding: 10px;
```

Comment:
note: `scoped` variables are not accessible elsewhere, so we can also use direct values.

Reply:
Ok, bhai, noted.